src/layouts/Home/components/Foundation.tsx

```diff
@@ -16,32 +15,29 @@ import MaxWidthLayout from 'layouts/MaxWidth';
 import { SectionCallout, StyledSectionHeader } from './SectionCallout';
 import { HomeLink } from './HomeLink';
 
+const imgStyles = {
+  width: 350,
+  minWidth: '100%',
+  maxWidth: '100%',
+  height: 266,
+  minHeight: '100%',
+  maxHeight: '100%',
+  zIndex: -1
+};
+
```

Comment:
Looks like these styles were extracted from the hero image and now incorrectly applied to the SVGs. Either way, I'm not sure they are needed.